lib/authlogic/acts_as_authentic/perishable_token.rb

```diff
@@ -58,7 +58,7 @@ def self.included(klass)
             extend ClassMethods
             include InstanceMethods
 
-            validates_uniqueness_of :perishable_token, if: :perishable_token_changed?
+            validates_uniqueness_of :perishable_token, if: :will_save_change_to_perishable_token?
```

Comment:
This change (in a `before_save`) is not strictly necessary, but is recommended for clarity?